site/docs/install-windows.md

```diff
@@ -42,16 +33,67 @@ Recommended: rename this binary to `bazel.exe` and move it to a directory on the
 
 Alternatively you can:
 
-*   [Install from Chocolatey](#install-using-chocolatey) (see below)
-*   [Install from Scoop](#install-using-scoop) (see below)
-*   [Build from source](install-compile-source.html)
+*   [Download Bazelisk](https://github.com/bazelbuild/bazelisk) instead of Bazel. Bazelisk is a
+    Bazel launcher that ensures you always use the latest Bazel release.
+*   [Install Bazel from Chocolatey](#install-using-chocolatey) (see below)
+*   [Install Bazel from Scoop](#install-using-scoop) (see below)
+*   [Build Bazel from source](install-compile-source.html)
+
+## 4. Optional: configure output directories
+
+**You can skip this step. Bazel can work without configuring the output directories, and will use
+its default values.**
+
+By default, Bazel writes to two directories:
+
+-   The "output user root", configurable with the `--output_user_root` flag.
+
+    This is where Bazel extracts from itself some embedded tools, its own Java runtime, and where it
```

Comment:
Nit: Hmm, do we need to mention the Java runtime? Some people may not like this, although it's not a really issue for them.

Reply:
Good idea. I'm not happy about omitting it (why should we hide it?), but I know it has been controversial (for questionable reasons). Removing.